templates/blog/newsletter-form.html

```diff
@@ -6,71 +6,21 @@ <h2>Newsletter <br>signup</h2>
 
   <div class="col-6 col-medium-4">
     <form action="https://ubuntu.com/marketo/submit" method="post" id="mktoForm_1212" onsubmit="ga('send', 'Newsletter', 'Signup', 'New homepage newsletter signup');">
-      <p>Select topics you're interested in:</p>
-
-      <ul class="p-list--divided">
-        <li class="p-list__item u-sv1">
-          <label class="p-checkbox">
-            <input class="p-checkbox__input" type="checkbox" aria-labelledby="insightscloudserver" name="insightscloudserver" value="true" />
-            <span class="p-checkbox__label" id="insightscloudserver">Cloud and Server</span>
-          </label>
-        </li>
-
-        <li class="p-list__item u-sv1">
+        <p>Get the latest Canonical news and updates in your inbox.</p>
```

Comment:
Might be good to make this a partial if it's the exact same form, that way any future changes only have to be made in one place